test/acceptance/tools/deploy_pep.py

```diff
@@ -225,7 +235,7 @@ def stop_pep(ip, user, password, port='22'):
         pid = sudo('ps -ef | grep "nodejs bin/pepProxy" | grep -v grep | awk \'{print $2}\'')
     else:
         raise NameError('Pep only can be started in Ubuntu and CentOS systems')
-    sudo('rm /tmp/pep/*')
+    sudo('rm -rf /tmp/pep')
```

Comment:
I use to perfer directories generated by mktemp to avoid conflicts, but not sure if makes sense in this case. Maybe there is a good reason to use `/tmp/pep`.